jetbrains-core/src/software/aws/toolkits/jetbrains/ui/wizard/java/SamInitProjectBuilderIntelliJ.kt

```diff
@@ -56,13 +58,16 @@ class SamInitModuleBuilder : ModuleBuilder() {
         rootModel.module.setModuleType(moduleType.id)
         val project = rootModel.project
 
-        template.samProjectTemplate.build(selectedRuntime, project.baseDir)
```

Comment:
This isnt quite right still,

it should be based on content entry, not the project base.

call `doAddContentEntry(rootModel)` and use the returned `ContentEntry.file` as the output Dir